changelog/@unreleased/pr-29.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: feature
+feature:
+  description: <!-- User-facing outcomes this PR delivers -->
```

Comment:
might want to clean this up

Reply:
Ah I think I figured this out now -- it's because of the PR template. Will update -- thanks!